client/mysqlimport.c

```diff
@@ -333,7 +333,7 @@ static int write_to_table(char *filename, MYSQL *mysql)
 #ifdef HAVE_SNPRINTF
     snprintf(sql_statement, FN_REFLEN*16+256, "DELETE FROM %s", tablename);
 #else
-    sprintf(sql_statement, "DELETE FROM %s", tablename);
+    snprintf(sql_statement, sizeof(sql_statement), "DELETE FROM %s", tablename);
```

Comment:
This is covered in the `#ifdef` above.

Reply:
In https://github.com/MariaDB/server/pull/2432#issuecomment-1459972727, you wrote:

> Just one part of my original review left. I recommend instead killing the sprintf and removing the snprintf guard for that one. Then I think we are in a good place.

Can you clarify this? I believe that you were asking @Chaloff to remove the `#ifdef HAVE_SNPRINTF` case altogether, and just leave the `#else` part. Is that correct?

Reply:
Assuming I interpreted that correctly, @Chaloff made that update in https://github.com/MariaDB/server/compare/33cac9ade64d460df168bea4d661f6c8a404df80..43fd6a3627d177865ffdb1eaa09d36100e6c5ed6